catapult/deploy.py

```diff
@@ -29,17 +29,26 @@
         "dry": "prepare a release without committing it",
         "yes": "automatic yes to prompt",
         "rollback": "needed to start a rollback",
+        "profile": "name of AWS profile to use",
     },
     default=True,
 )
 @utils.require_2fa
 def start(
-    _, name, env, version=None, bucket=None, dry=False, yes=False, rollback=False
+    _,
+    name,
+    env,
+    version=None,
+    bucket=None,
+    dry=False,
+    yes=False,
+    rollback=False,
+    profile=None,
```

Comment:
Is there a better way to add a common argument than copy/pasting it (and its help text) everywhere?

Reply:
I think you can do something using [Config](http://docs.pyinvoke.org/en/stable/concepts/configuration.html), but I've never used it